components/system/components/CodeBlock.js

```diff
@@ -3,22 +3,63 @@ import * as Constants from "~/common/constants";
 
 import { css } from "@emotion/react";
 
-const STYLES_CODE_BLOCK = css`
-  font-family: ${Constants.system.mono};
-  background-color: ${Constants.system.black};
-  color: ${Constants.system.white};
-  border-radius: 4px;
-  padding: 24px;
-  font-size: 14px;
-  word-wrap: break-word;
-  white-space: pre-wrap;
-  width: 100%;
+const STYLE_CODE_BLOCK = css`
+  background-color: ${Constants.system.pitchBlack};
+  color: ${Constants.system.white}:
+  font-size: 16px;
+  border-color: ${Constants.system.yellow};
+  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.50);
+  border-radius: 7px;
+  padding: 10px 10px;
 `;
 
-export const CodeBlock = (props) => {
-  return (
-    <div css={STYLES_CODE_BLOCK}>
-      <code {...props} />
-    </div>
-  );
-};
+const STYLE_PADDING = css`
```

Comment:
Add `white-space: pre-wrap`

Reply:
Oh, this fixed the odd flattening of the codeblocks ! 😍

Reply:
:)